scripts/ci/lib.sh

```diff
@@ -502,6 +502,7 @@ mark_collector_release() {
     export CIRCLE_PULL_REQUEST="https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/${JOB_NAME}/${BUILD_ID}"
```

Comment:
In the current version, these `export`s get propagated through to the invoking environment, but no longer will after the `(...)` change. This should be perfectly fine, but if after the suggested change you see errors about these vars not being defined, the behavior change might be to blame.

Reply:
I think the exports are only needed for `create_update_pr.sh` cause it was written to run in CircleCI and took those for granted. So the exports are harmless her, but a proper fix would be to eliminate them by making `create_update_pr.sh` CI-independent.